NEWS.md

```diff
@@ -1,6 +1,8 @@
 # wwinference
 
 This will serve as our change-log
+
+- 2024-08-23: Added new `wwinference_fit` class with corresponding print and summary methods.
```

Comment:
I've been putting them in chronological order top to bottom!

Reply:
Got it! That said, I think the NEWS.md file is improperly structured. I will put it on the bottom and then make an issue pointing this out (R CMD check highlights this).